qai/qai_plugin.py

```diff
@@ -191,6 +192,18 @@ def roll(self, mask, target, args):
         """
         return f'{mask.nick} rolls {random.randint(0, 100)}!'
 
+    @command(name='8ball')
+    @channel_only
+    @nickserv_identified
+    def eight_ball(self, mask, target, args):
+        """Ask the mysterious 8ball a question.
+
+            %%8ball WORDS ...
+        """
+        if self.spam_protect('8ball', mask, target, args):
+                return
```

Comment:
i trusted in you!
you should be ashamed :(

Reply:
xDD nice
its python tho so works anyway!